_includes/doc-side-concepts-nav.html

```diff
@@ -1,22 +1,39 @@
+            <li><a href="{{ site.baseurl }}/docs/concepts/repository.html" {% if current[3] == 'repository.html' %}class='current'{% endif %}>Repository</a></li>
+            <li><a href="{{ site.baseurl }}/docs/concepts/snapshot.html" {% if current[3] == 'snapshot.html' %}class='current'{% endif %}>Snapshot</a></li>
+        </ul>
+    </li>
+    <li>
+        <ul id="doc-side-nav-inside">
+            <li><h6 class="doc-side-nav-sub-title">Messages</h6></li>
+            <li><a href="{{ site.baseurl }}/docs/concepts/command.html" {% if current[3] == 'command.html' %}class='current'{% endif %}>Command</a></li>
+            <li><a href="{{ site.baseurl }}/docs/concepts/event.html" {% if current[3] == 'event.html' %}class='current'{% endif %}>Event</a></li>
+            <li><a href="{{ site.baseurl }}/docs/concepts/rejection.html" {% if current[3] == 'rejection.html' %}class='current'{% endif %}>Rejection</a></li>
+            <li><a href="{{ site.baseurl }}/docs/concepts/command-handler.html" {% if current[3] == 'command-handler.html' %}class='current'{% endif %}>Command Handler</a></li>
+            <li><a href="{{ site.baseurl }}/docs/concepts/event-subsriber.html" {% if current[3] == 'event-subsriber.html' %}class='current'{% endif %}>Event Subscriber</a></li>
+        </ul>
+    </li>
+    <li>
+        <ul id="doc-side-nav-inside">
+            <li><h6 class="doc-side-nav-sub-title">Architecture</h6></li>
+            <li><a href="{{ site.baseurl }}/docs/concepts/event-store.html" {% if current[3] == 'event-store.html' %}class='current'{% endif %}>Event Store</a></li>
+            <li><a href="{{ site.baseurl }}/docs/concepts/aggregate-mirror.html" {% if current[3] == 'aggregate-mirror.html' %}class='current'{% endif %}>Aggregate Mirror</a></li>
+            <li><a href="{{ site.baseurl }}/docs/concepts/bounded-context.html" {% if current[3] == 'bounded-context.html' %}class='current'{% endif %}>Bounded Context</a></li>
+            <li><a href="{{ site.baseurl }}/docs/concepts/command-bus.html" {% if current[3] == 'command-bus.html' %}class='current'{% endif %}>Command Bus</a></li>
+            <li><a href="{{ site.baseurl }}/docs/concepts/command-store.html" {% if current[3] == 'command-store.html' %}class='current'{% endif %}>Command Store</a></li>
+            <li><a href="{{ site.baseurl }}/docs/concepts/event-bus.html" {% if current[3] == 'event-bus.html' %}class='current'{% endif %}>Event Bus</a></li>
+            <li><a href="{{ site.baseurl }}/docs/concepts/query-service.html" {% if current[3] == 'query-service.html' %}class='current'{% endif %}>Query Service</a></li>
+            <li><a href="{{ site.baseurl }}/docs/concepts/integration-event.html" {% if current[3] == 'integration-event.html' %}class='current'{% endif %}>Integration Event</a></li>
+            <li><a href="{{ site.baseurl }}/docs/concepts/command-service.html" {% if current[3] == 'command-service.html' %}class='current'{% endif %}>Command Service</a></li>
```

Comment:
Please reorder the items:
1. Bounded Context
2. Integration Event
3. Command Bus
4. Command Store
5. Command Service
6. Event Bus
7. Event Store
8. Aggregate Mirror
9. Query Service